### ISSUE TITLE
Issue with pkgdown sites

### ISSUE THREAD

**Issue by Kertoo** [wviechtb/mathjaxr#16]
Hej,

I use `mathjaxr` as a LaTeX backend in [my package](https://github.com/ncn-foreigners/singleRcapture) and it usually works really good in standard manual. The problem is that a few months back with the update 2.1.0 to `pkgdown` the math on GitHub pages stopped rendering properly (for example in in [here](https://ncn-foreigners.github.io/singleRcapture/reference/estimatePopsize.html)). This issue is not limited to `mathjaxr`, people have discussed it for example in https://github.com/r-lib/pkgdown/issues/2704. 

I think it doesn't recognized the `\loadmathjax` macro anymore, this is one of the warning messages I get while trying to set up the website:

```
In tools::parse_Rd(path, encoding = "UTF-8") :
  ./man/controlPopVar.Rd:81: unknown macro '\loadmathjax'
```

Is there any fix to this issue? Could one define this macro in for example `_pkgdown.yml`?

**Comment by wviechtb** [comment]
Yeah, I ran into the same issue. The last version of `pkgdown` accidentally disabled macro loading (see https://github.com/r-lib/pkgdown/issues/2782). This has been fixed, but you will have to install the current development version of `pkgdown` (or an older one).
